tests/unit/test_main.py

```diff
@@ -246,5 +248,69 @@ def test_preference_args(self):
         prefs = main.preference(None, prefs_args)
         self.assertEquals(len(prefs), 0)
 
+
+class TestCheckMozregresionVersion(unittest.TestCase):
+    @patch('requests.get')
+    @patch('sys.stdout')
+    def test_pypi_is_accesible(self, stdout, get):
```

Comment:
Hmm, I don't understand the name of this function. Shouldn'it be something like "test_version_is_up_to_date" ?